dora/core/client/fs/src/main/java/alluxio/client/block/stream/DefaultBlockWorkerClient.java

```diff
@@ -140,6 +140,19 @@ public DefaultBlockWorkerClient(UserState userState, GrpcServerAddress address,
     mTracker = DETECTOR.track(this);
```

Comment:
Can we replace the above 6 lines of statements with calling the other constructor:
```
DefaultBlockWorkerClient(userState, address, alluxioConf, mStreamingChannel, mRpcChannel);
```
to avoid duplicated code?

Reply:
One way is to put those into init function, both call it.
Will replace in a following pr.